### PR TITLE
BCS-01C Fix typos and increase consistency in comments

### PR DESCRIPTION
**Pull request by hswopeams**
This PR fixes typos and increases consistency in some of the NatSpec blocks across files.

### CODE REVIEW

File: contracts/interfaces/diamond/IDiamondCut.sol [hunk 1]
```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.9;
 /**
  * @title IDiamondCut
  *
- * @notice Diamond Facet management
+ * @notice Manges Diamond Facets.
```
Comment (zajck) [inline contracts/interfaces/diamond/IDiamondCut.sol:7]:
```suggestion
 * @notice Manages Diamond Facets.
```

**Review by cliffhall — DISMISSED** [review]
LGTM! 👍

File: contracts/interfaces/diamond/IDiamondCut.sol (hunk 1 above)
Comment (hswopeams) [reply contracts/interfaces/diamond/IDiamondCut.sol]: Fixed

**Review by cliffhall — APPROVED** [review]
LGTM! 👍

**Review by zajck — APPROVED** [review]
LGTM

**Pull request merged**